cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,79 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SettingsPageObject from "../support/pages/settings.pageObject";
+import homePageObject from "../support/pages/home.pageObject";
+import SignInPageObject from "../support/pages/signIn.pageObject";
+import faker from "faker";
+
+const settingsPage = new SettingsPageObject;
+const homePage = new homePageObject;
+const signInPage = new SignInPageObject;
+
 describe('Settings page', () => {
-  before(() => {
+  let user;
+  const newUser = {
+    username: faker.name.firstName().toLowerCase(),
+    bio: faker.lorem.words(),
+    email: faker.internet.email().toLowerCase(),
+    password: faker.internet.password()
+  }
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.register(user.email, user.username, user.password);
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
 
+    settingsPage.updateUsernameField(newUser.username);
+    settingsPage.clickOnUpdateBtnSet();
+
+    cy.getByDataCy('username-profile').contains(newUser.username);
+
   });
 
   it('should provide an ability to update bio', () => {
 
+    settingsPage.updateBioField(newUser.bio);
+    settingsPage.clickOnUpdateBtnSet();
+
+    cy.getByDataCy('bio-profile').contains(newUser.bio);
```

Comment:
try to create assertion method here as well

Reply:
changed